ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -477,6 +477,14 @@ FAN_EFHLC_COCDSP_NOCDSP:
   - COCDSP
   - NOCDSP
 
+FAN_VSC_SCRSS:
+  description: "Fan with scrubber start/stop and status."
+  is_canonical: true
+  implements:
+  - FAN
+  - VSC
```

Comment:
can you please add SS? we usually call out SS and VSC separately. 

please make this change and we can approve the PR.

Reply:
Done